docs/en/trust.rst

```diff
@@ -631,6 +631,17 @@ Below is a non-normative example of a Trust Chain in its original format (JSON A
 .. note::
 
     The entire Trust Chain is verifiable by only possessing the Trust Anchor's public keys.
+There are events where keys are unavailable to verify the entire trust chain. These cases can arise as follows:
```

Comment:
false, since the historical key endpoint published by trust anchors, always provides the historical keys needed to validate the historical trust chains